.travis.yml

```diff
@@ -25,6 +17,21 @@ jobs:
       addons:
         srcclr: true
 
+    - stage: 'Integration tests'
+      cache: false
```

Comment:
can you please add why you added this one?

Reply:
Copy pasted from javascript SDK ([here](https://github.com/optimizely/javascript-sdk/blob/8eedd63296fc722e737dd7f7a0ac77e43018a8c8/.travis.yml#L50)), i dont know travis CI much.. didnt want to take any risks making modifications on my own :)